### PR TITLE
ADR - version serialization schemes

### PR DESCRIPTION
**Pull request by JaredCorduan**

### CODE REVIEW

File: docs/adr/2022-12-05_002-version-serialization-schemes.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Decision
+
+Create a new typeclass to replace ToCBOR and FromCBOR which depends not only on the type
+but also on the protocol version (or, alternatively, the ledger era).
+The first instances will all be implemented identically to our current
+ToCBOR and FromCBOR instances for all protocol versions up to the current one.
+
+## Consequences
+
+This change effects downstream components that rely on the ledger serializers,
```
Comment (teodanciu) [inline docs/adr/2022-12-05_002-version-serialization-schemes.md]: Perhaps `affects` would be better ?

File: docs/adr/2022-12-05_002-version-serialization-schemes.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+tags: [Accepted]
+---
+
+## Status
+
+Accepted
+
+## Context
+
+The cardando ledger used CBOR to serialize the blocks and transaction on the network.
```
Comment (teodanciu) [inline docs/adr/2022-12-05_002-version-serialization-schemes.md]: Maybe `blocks and transactions` ?

**Review by teodanciu — APPROVED** [review]
Looks good to me, nicely explained.

File: docs/adr/2022-12-05_002-version-serialization-schemes.md (hunk 1 above)
Comment (JaredCorduan) [reply docs/adr/2022-12-05_002-version-serialization-schemes.md]: :facepalm: :laughing: you'd think I could get that one correct by now... thank you for saving me future embarrassment! :grin:

**Pull request merged**